netbox/project-static/bundle.js

```diff
@@ -14,24 +17,49 @@ const options = {
 // Get CLI arguments for optional overrides.
 const ARGS = process.argv.slice(2);
 
+function copyFiles(files) {
+    return Promise.all(files.map(f => {
+       return copyFilePromise(f.source, f.dest);
+    }));
+}
+
 async function bundleGraphIQL() {
-  try {
-    const result = await esbuild.build({
-      ...options,
-      entryPoints: {
-        graphiql: 'netbox-graphiql/index.ts',
-      },
-      target: 'es2016',
-      define: {
-        global: 'window',
-      },
-    });
-    if (result.errors.length === 0) {
-      console.log(`✅ Bundled source file 'netbox-graphiql/index.ts' to 'graphiql.js'`);
+  fileMap = [
+    {
+      source: './node_modules/react/umd/react.production.min.js',
+      dest: './dist/react.production.min.js'
```

Comment:
Can we move these to a subdirectory ( e.g. `static/graphiql/`) to keep them isolated from "native" static resources?